### PR TITLE
Review and fix tools docs

### PR DESCRIPTION
**Pull request by JuanAr**
Fixes several broken links.

## Proposed Changes
  - Fixed all links broken after moving tools to `packages` folder due to `lerna` integration
  - Fixed several typos

## Testing
No broken links should appear in docs

### CODE REVIEW

File: packages/MSBot/.vscode/launch.json [hunk 1]
```diff
@@ -1,161 +0,0 @@
...
-                "--authoringKey",
-                "2f510b5e-10fe-4f53-9159-b134539ac594",
-                "--subscriptionKey",
-                "2f510b5e-10fe-4f53-9159-b134539ac594",
-                "--version",
-                "1.0"
-            ]
-        }
-    ]
-}
```
Comment (tomlm) [inline packages/MSBot/.vscode/launch.json]: please do not remove, these are not real keys

File: packages/MSBot/.vscode/tasks.json [hunk 2]
```diff
@@ -1,18 +0,0 @@
...
-            "problemMatcher": [
-                "$tsc"
-            ],
-            "group": {
-                "kind": "build",
-                "isDefault": true
-            }
-        }
-    ]
-}
```
Comment (tomlm) [inline packages/MSBot/.vscode/tasks.json]: do not remove

File: packages/LUIS/.vscode/tasks.json [hunk 3]
```diff
@@ -1,14 +0,0 @@
...
-    "tasks": [
-        {
-            "type": "npm",
-            "script": "build",
-            "problemMatcher": [
-                "$tsc"
-            ]
-        }
-    ]
-}
```
Comment (tomlm) [inline packages/LUIS/.vscode/tasks.json]: do not remove

File: packages/LUIS/.vscode/launch.json [hunk 4]
```diff
@@ -1,17 +0,0 @@
...
-            "type": "node",
-            "request": "launch",
-            "name": "Launch Program",
-            "program": "${workspaceFolder}\\bin\\luis",
-            "args" :[
-                "update", "settings", "--public", "true"
-            ]
-        }
-    ]
-}
```
Comment (tomlm) [inline packages/LUIS/.vscode/launch.json]: do not remove

File: packages/Chatdown/.vscode/launch.json [hunk 5]
```diff
@@ -1,23 +0,0 @@
...
-                "tdd",
-                "--timeout",
-                "999999",
-                "--colors",
-                "${workspaceFolder}/test/chatdown.lib.test.suite.js"
-            ],
-            "internalConsoleOptions": "openOnSessionStart"
-        },
-    ]
-}
```
Comment (tomlm) [inline packages/Chatdown/.vscode/launch.json]: do not remove

File: packages/.vscode/tasks.json [hunk 6]
```diff
@@ -1,15 +0,0 @@
...
-        {
-            "type": "npm",
-            "script": "build",
-            "path": "LUIS/",
-            "problemMatcher": [
-                "$tsc"
-            ]
-        }
-    ]
-}
```
Comment (tomlm) [inline packages/.vscode/tasks.json]: do not remove

File: .vscode/tasks.json [hunk 7]
```diff
@@ -1,15 +0,0 @@
...
-        {
-            "type": "npm",
-            "script": "build",
-            "path": "packages/MSBot/",
-            "problemMatcher": [
-                "$tsc"
-            ]
-        }
-    ]
-}
```
Comment (tomlm) [inline .vscode/tasks.json]: do not remove

**Comment by tomlm** [comment]
> 

do not remove

---
Refers to: .vscode/launch.json:20 in 4b62cdd. [](commit_id = 4b62cdd15fc0587675b26f07bf2e7255694f5ae1, deletion_comment = True)

**Comment by JuanAr** [comment]
Just kept only .md file updates.

**Pull request merged**